README.md

```diff
@@ -61,9 +61,9 @@ Adding a health check to an app
         ...
     ```
 
-2. Initialise any clients that have `Checker` type functions you wish to use
+3. Initialise any clients that have `Checker` type functions you wish to use
```

Comment:
Is there an example that can be used here? Most other steps have examples but not this one.

Reply:
I will ad an example here